<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="landingLayout" native="true">
   <property name="geometry">
    <rect>
     <x>-420</x>
     <y>-570</y>
     <width>1000</width>
     <height>1400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#landingLayout {
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(141, 176, 255, 255), stop:1 rgba(255, 236, 202, 255));
}</string>
   </property>
   <widget class="QFrame" name="applicationName">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>640</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Billeasy</string>
     </property>
    </widget>
    <widget class="QLabel" name="subtitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>The Application</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>790</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="container">
     <item>
      <widget class="QLabel" name="usernameLbl">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Crash Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameIpt">
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding: 7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="passwordLbl">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
margin-top: 15px;</string>
       </property>
       <property name="text">
        <string>Crash Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordIpt">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding: 7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rememberChk">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
margin-top: 15px;</string>
       </property>
       <property name="text">
        <string>Remember Me</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
	border-radius: 8px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	background: rgba(255, 255, 255, 0.3);
	padding: 20px;
}

*:hover {
	background: rgba(255, 255, 255, 0.7);
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="errorLbl">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>732</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgba(255, 0, 0, 0.5);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
